lib/foreman_discovery/proxy_operations.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanDiscovery
+  class ProxyOperations < ::ProxyAPI::Resource
+    def initialize args
+      @url  = args[:url] + "/#{args[:operation]}"
+      super args
+    end
+
+    def parse_put_operation
+      # parse put did not work from here, until problem is fixed using rest client directly 
+      RestClient.put @url, {}, :content_type => :json, :accept => :json
```

Comment:
does anyone use parse put?
maybe I could try to figure it out from a usage example